src/api/Alchemy.ts

```diff
@@ -0,0 +1,94 @@
+import { ChainId } from '@dcl/schemas'
+import API from 'decentraland-gatsby/dist/utils/api/API';
+
+export type TokenBalanceResponse = {
+  contractAddress: string,
+  tokenBalance: string | null,
+  error: string | null,
+}
+
+export type TokenBalancesResponse = {
+  id: number,
+  jsonrpc: string,
+  result:{
+    address: string,
+    tokenBalances: TokenBalanceResponse[],
+  },
+}
+
+export type BalanceResponse = {
+  id: number,
+  jsonrpc: string,
+  result: string,
+}
+
+export type BlockNumberResponse = {
+  id: number,
+  jsonrpc: string,
+  result: string,
+}
+
+export class Alchemy extends API {
+
+  static Cache = new Map<string, Alchemy>()
+
+  static from(baseUrl: string) {
+    if (!this.Cache.has(baseUrl)) {
+      this.Cache.set(baseUrl, new this(baseUrl))
+    }
+
+    return this.Cache.get(baseUrl)!
+  }
+
+  static get(chain: ChainId) {
+    switch (chain) {
+      case ChainId.ETHEREUM_MAINNET:
+        return this.from('https://eth-mainnet.alchemyapi.io')
+      case ChainId.MATIC_MAINNET:
+        return this.from('https://polygon-mainnet.g.alchemy.com')
+    }
+    throw new Error('Unsuported chain id')
+  }
+
+  async getTokenBalances<P extends keyof TokenBalancesResponse>(address: string, tokens: string[]) {
+    return await this.fetch<TokenBalancesResponse>(
+      '/v2/HcW-dJIcedLgGAB8htL7M1L8ir2Sihfe',
```

Comment:
isn't `HcW-dJIcedLgGAB8htL7M1L8ir2Sihfe` and apikey? if it is anyone cloud copy it and outlimit our requests

wouldn't be better use a server request for that?

(I'm asking cause I don't know how alchemy works)